src/theme/components/socialmedialinks.ts

```diff
@@ -42,73 +42,77 @@ function getColor(color) {
+      flexDirection: { base: labelsOn ? "column" : "row", md: layout },
+      justifyContent: { base: "center", md: "unset" },
       flexWrap: "wrap",
-      minWidth: { base: "44px", md: "unset" },
-      minHeight: { base: "44px", md: "unset" },
-      fontStyle: "normal",
-      fontSize: "desktop.body.body2",
-      textDecoration: "none",
-      color: "ui.typography.heading",
-      _hover: {
-        color: "ui.typography.heading",
-        textDecoration: "underline 1px dotted",
+      gap: "s", // 1rem / 16px
+      marginBottom: "unset",
+      li: {
+        marginInlineEnd: "0",
+        marginTop: "0",
+        width: { base: labelsOn ? "100%" : "unset", md: "unset" },
+        _notFirst: {
+          marginTop: "0",
+        },
       },
-      _visited: {
+      a: {
+        display: "flex",
+        justifyContent: "center",
+        alignContent: "center",
+        flexWrap: "wrap",
+        minWidth: { base: "44px", md: "unset" },
+        minHeight: { base: "44px", md: "unset" },
+        fontStyle: "normal",
+        fontSize: "desktop.body.body2",
+        textDecoration: "none",
         color: "ui.typography.heading",
-      },
-      _dark: {
-        color: "dark.ui.typography.heading",
         _hover: {
-          color: "dark.ui.typography.heading",
+          color: "ui.typography.heading",
+          textDecoration: "underline 1px dotted",
         },
         _visited: {
+          color: "ui.typography.heading",
+        },
+        _dark: {
           color: "dark.ui.typography.heading",
+          _hover: {
+            color: "dark.ui.typography.heading",
+          },
+          _visited: {
+            color: "dark.ui.typography.heading",
+          },
         },
       },
-    },
-    div: {
-      display: "flex",
-      alignItems: "center",
-      gap: "xs", // .5rem / 8px
-      alignSelf: "stretch",
-      justifyContent: "center",
-      width: "fit-content",
-      height: "fit-content",
-    },
-    svg: {
-      fill: "ui.typography.heading",
-      _dark: {
-        fill: "dark.ui.typography.heading",
+      div: {
+        display: "flex",
+        alignItems: "center",
+        gap: "xs", // .5rem / 8px
+        alignSelf: "stretch",
+        justifyContent: "center",
+        width: "fit-content",
+        height: "fit-content",
       },
-    },
+      svg: {
+        fill: "ui.typography.heading",
+        _dark: {
+          fill: "dark.ui.typography.heading",
+        },
+      },
+    };
   }),
   // The variants are based on border values
   variants: {
     // Each variant is a function that returns an object with styles for that variant.
     straight(props: StyleFunctionProps) {
-      const { color, layout } = props;
+      const { color, labelsOn } = props;
       let theseColors = getColor(color);
       return {
-        flexDirection: { base: "column", md: layout },
         gap: "xs", // Borders require the gap to reduce from 1rem to .5rem / 8px.
         a: {
```

Comment:
Because of the min-width and min-height attributes that are being applied for mobile viewport, we need to add a few more styles to get the outline to fill the full space.  Add the following within the `a` object:
```
".platLink": {
  minHeight: {
    base: "44px",
    md: "0",
  },
  minWidth: {
    base: "44px",
    md: "0",
  },
},
```